Header header
int32 user_id
float32[] position
float32[] joint_angle

